wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -959,18 +951,11 @@ extension ZMUserSession: ZMSyncStateDelegate {
                 return (selfClient?.qualifiedClientID, hasRegisteredMLSClient)
             }
 
-            if hasRegisteredMLSClient {
-                await performsMLSClientUpdates()
-            } else {
-                // If we discover that
-                // there are MLS public keys on the backend, the MLS feature is enabled and there is no registered MLS
-                // client,
-                // we should create one.
-                let needsToRegisterMLSClient = BackendInfo.isMLSEnabled && mlsFeature.isEnabled
-                if let qualifiedSelfClientID, needsToRegisterMLSClient {
-                    await createMLSClient(qualifiedID: qualifiedSelfClientID)
-                    await performsMLSClientUpdates()
-                }
+            if let qualifiedSelfClientID {
```

Comment:
And if there is no qualified self client id? Is this an expected state? What should we do?

Reply:
No, it's not expected. I would add logs, but wouldn't terminate the app.